json-ld/src/test/scala/io/renku/jsonld/JsonLDSpec.scala

```diff
@@ -455,6 +460,246 @@ class JsonLDSpec extends AnyWordSpec with ScalaCheckPropertyChecks with should.M
     }
   }
 
+  "flatten" should {
```

Comment:
I think we can get rid of these tests as flatten was moved to the Flattener class